packages/cosmic-swingset/src/kernel-stats.js

```diff
@@ -56,10 +63,10 @@ const recordToKey = record =>
 /**
  * @param {{
  *   metricMeter: import('@opentelemetry/sdk-metrics-base').Meter,
- *   labels: Record<string, string>
+ *   attributes: import('@opentelemetry/api-metrics').Attributes,
```

Comment:
Looks like [this type](https://github.com/open-telemetry/opentelemetry-js/blob/f59c5b268bd60778d7a0d185a6044688f9e3dd51/experimental/packages/opentelemetry-api-metrics/src/types/Metric.ts#L108) is no more optional than `Record`.